components/unarchive_channel_modal/unarchive_channel_modal.test.jsx

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Modal} from 'react-bootstrap';
+
+import {browserHistory} from 'utils/browser_history';
+
+import UnarchiveChannelModal from './unarchive_channel_modal';
+
+describe('components/unarchive_channel_modal', () => {
+    function emptyFunction() {} //eslint-disable-line no-empty-function
```

Comment:
Instead of making an empty function, you can use `jest.fn()` to make ones for you without causing an ESLint warning.